Extension/src/LanguageServer/references.ts

```diff
@@ -137,10 +142,11 @@ export class ProgressHandler {
         this.prevVisibleRangesLength = visibleRangesLength;
     }
 
-    public reportProgress(progress: vscode.Progress<{message?: string; increment?: number }>, forceUpdate: boolean, isPeek: boolean): void {
-        const helpMessage: string = isPeek ? "" : ` ${localize("click.search.icon", "To preview results, click the search icon in the status bar.")}`;
+    public reportProgress(progress: vscode.Progress<{message?: string; increment?: number }>, forceUpdate: boolean): void {

```

Comment:
>public [](start = 4, length = 6)

this can be private, it's only called in handleProgress(...)

Reply:
Maybe I'll check.

Reply:
Fixed